### ISSUE TITLE
Add leiningen tab for dependencies.

### ISSUE THREAD

**Issue by hepin1989** [akka/akka#26237]
Add leiningen for clojure users.
A friend is using Akka from Clojure:https://zhuanlan.zhihu.com/p/54722983
![image](https://user-images.githubusercontent.com/501740/51073268-3c644880-16a9-11e9-9990-08e642c38460.png)

**Comment by hepin1989** [comment]
@MarchLiu cc

**Comment by MarchLiu** [comment]
I will try it:)

**Comment by patriknw** [comment]
You want to add a forth tab for Clojure dependencies?
I'm not sure that we want that. That would indicate that we fully support Clojure, which isn't the case. Great if it works on Clojure but it's not something we can support, document, test and maintain.

Translating the dependencies must be the least of efforts when trying to use Akka from Clojure.

**Comment by hepin1989** [comment]
Yes, @MarchLiu is advising how to make using of Akka in Clojure, he was tech lead at huobi :). add a dependency tab show a good welcome to the Clojure community.
